### PR TITLE
Enables Fleet detectors on for D&S blade.

### PR DESCRIPTION
**Pull request by Ealianis**
## Overview
Added the config files for new resource type `containerservice/fleets`, enabling 

## Related Work Item
[ADO Task](https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/16713931)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist <span>&#9989;</span>
- [x] I have looked over the diffs.
- [x] I have run the linter to catch any errors.
- [x] There are no errors from my changes on this branch.
- [x] I have requested review on this PR.

## Extra Notes
PR will remain open as we discuss the quality of the detector that this is mapped to. It appears there is a design around a detector being a category of issues vs. a specific issue.

### CODE REVIEW

**Review by nmallick1 — CHANGES_REQUESTED** [review]

File: AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/fleets/config.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "homePageText": {
+        "title": "Azure Kubernetes Fleet Diagnostics (Preview)",
+        "description": "Investigate how your fleets are performing, diagnose issues, and discover how to improve its reliability.",
+        "searchBarPlaceHolder": "Search a keyword that best describes your issue"
+    },
+	"matchRegEx": "/Microsoft.ContainerService/managedClusters/",
```
[the remark this answers is not in the document]
Comment (Ealianis) [reply AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/fleets/config.json]: Fixed.

File: AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/fleets/config.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "homePageText": {
+        "title": "Azure Kubernetes Fleet Diagnostics (Preview)",
+        "description": "Investigate how your fleets are performing, diagnose issues, and discover how to improve its reliability.",
+        "searchBarPlaceHolder": "Search a keyword that best describes your issue"
+    },
+	"matchRegEx": "/Microsoft.ContainerService/fleets/",
+	"searchSuffix": "Fleet",
+	"azureServiceName": "Azure Kubernetes Fleet",
+	"armApiConfig": {
+		"armApiVersion": "2019-04-01"
```
Comment (Ealianis) [inline AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/fleets/config.json:11]: @nmallick1 , does this API version reference the resource type's most recent API version?

Comment (nmallick1) [reply AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/fleets/config.json:11]: Yes, this is the ARM API version corresponding to the RP that has the /detectors and /detectors/{detectorId} API implementation.

Comment (Ealianis) [reply AngularApp/projects/app-service-diagnostics/src/armResourceConfig/microsoft.containerservice/fleets/config.json]: Updated

**Pull request merged**